gestion_base_adresse/test/test_execute_functions.py

```diff
@@ -38,7 +38,7 @@ def test_calcul_num_adr(self):
         # numérotation inversée, donc nombre pair à gauche
 
         # Suppression des points pour pouvoir tout tester
-        sql = "delete from adresse.point_adresse"
+        sql = "TRUNCATE TABLE adresse.point_adresse"
```

Comment:
```suggestion
        sql = "TRUNCATE TABLE adresse.point_adresse" RESTART IDENTITY
```
Il faut ajouter le `RESTART IDENTITY`